docs/Timers/index.md

```diff
@@ -0,0 +1,155 @@
+
+The timers Community Framework is providing will offer improved performance by removing unneccessary function calls and removing the use dynamic arrays. As testing has shown with [Expression's](../ExpressionVM/index.md) there is a huge performance gain when using dual Linked Lists over even static arrays. 
+
+All timers have safeguards built in, such as with [Instance Timers](#Instance-Timers) only allowing to be ran in `Managed` classes. 
+
+## Instance Timers
+
+Instance Timers can only be ran in `Managed` classes. This benefit provides safe memory management and allows for timers to be deleted when they are no longer required.
+
+### Basic Example
+
+```csharp
+class SomeClass : Managed
+{
+    void SomeClass()
+    {
+        float interval = 0.5; // The timer will tick every 0.5 seconds.
+        CF_Timer.Create(this, "OnTick", interval);
+    }
+
+    void OnTick(CF_TimerBase timer, float deltaTime)
+    {
+        Print("Timer has ran for " + timer.m_TimerElapsed + " seconds.");
+    }
+};
+```
+
+Memory management is handled all for you. To create a timer you run `CF_Timer.Create` with the arguments you specify. Note that an interval of `0.0` means the timer will tick every frame. 
+
+### Advanced Example
+
+```csharp
+class Menu : UIScriptedMenu
+{
+    static const float FADE_TIME = 0.5;
+
+    void Menu()
+    {
+        CF_Timer.Create(this, "FadeIn", 0.0, new CF_TimerParam1<Widget, float>(GetLayoutRoot(), FADE_TIME));
+    }
+
+    void FadeIn(CF_TimerBase timer, float deltaTime, Widget widget, float timeToFade)
+    {
+		float alpha = timer.m_TimeElapsed / timeToFade;
+        widget.SetAlpha(Math.Clamp(alpha, 0, 1));
+
+        if (timer.m_TimeElapsed >= timeToFade)
+        {
+            timer.Stop();
+        }
+    }
+};
+```
+
+## Generic Timers
+
+Generic Timers are used in places where you want to use the same code for the timer in multiple places at once, or you want to use the timer outside of a `Managed` class or in a static/global function.
+
+### Basic Example
+
+```csharp
+class SomeClass : Managed
+{
+    void SomeClass()
+    {
+        new SomeTimer(this);
+    }
+};
+
+class SomeTimer : CF_TimerBase
+{
+    SomeClass m_SomeClass;
+
+    void SomeTimer(SomeClass someClass)
+    {
+        m_SomeClass = someClass;
+
+        m_Interval = 0.5;
+        
+        // Start this timer automatically.
+        Start();
+    }
+
+    override void OnTick(float dt)
+    {
+        Print("Timer has ran for " + m_TimerElapsed + " seconds and the owner is " + m_SomeClass + ".");
+
+        // Experiment removing the `Managed` class from `SomeClass` inheritance.
+        if (m_SomeClass == null)
+        {
+            Destroy();
+        }
+    }
+};
+```
+
+### Advanced Example
+
+Comparing to the advanced example from `Instance Timers`, this is how the new code will be created and ran.
```

Comment:
I do not understand the difference to the other example. Whatever you mean with "Instance Timers" is not self-explanatory from the example below.